lib/CabalToDhall.hs

```diff
@@ -124,43 +129,107 @@ data KnownDefault
   deriving ( Bounded, Enum, Eq, Ord, Read, Show )
 
 
-data PreludeReference
+data Reference
```

Comment:
I thought in separate references to prelude from types but i was not sure how to refactorize the functions that uses them. 
It seems that is better to centralize the dependency to the `dhall-to-cabal` prelude to make it the code more resilient to layout changes. However i've not removed the rest of snall references to `prelude` itself